build.sh

```diff
@@ -86,9 +84,38 @@ then
   $cmd
 elif [ -z "$BUILD" ]
 then
+  mode="-Dfailsafe.forkCount=$processors -Dcom.vaadin.testbench.Parameters.testsInParallel=$parallel"
   ### Run IT's in merged module
-  cmd="mvn verify -Drun-it -Drelease -Dcom.vaadin.testbench.Parameters.testsInParallel=1 $args -pl integration-tests"
-  tcLog "Running merged ITs - mvn verify -Drun-it -Drelease -pl integration-tests ..."
+  cmd="mvn verify -B -Drun-it -Drelease $mode $args -pl integration-tests"
+  tcLog "Running merged ITs - mvn verify -B -Drun-it -Drelease -pl integration-tests ..."
   echo $cmd
-  $cmd
+  $cmd 2>&1 | egrep --line-buffered -v \
```

Comment:
Won't suppressing the output make the build harder to debug?

Reply:
Failed tests are run a second time, there we have the full output if it fails again.
Removed stuff in this grep is just for removing lines that does not say so much, but just noise.